docs/tutorials/sharing-environments.md

```diff
@@ -47,7 +47,7 @@ $ git add .flox
 $ git commit -m "sharing flox environment"
 ```
 
-Another developer on the same project can get started immediately with [`flox activate`][flox_activate]
+Another developer on the same project can get started immediately with [`flox activate`][flox_activate] to access exactly the same environment, including the defined packages:
```

Comment:
I propose:

> Another developer on the same project but on a different machine that has no packages installed yet can get started immediately with [`flox activate`][flox_activate] which will implicitly download and install any missing packages:

Reply:
Thanks for the suggestion! "install" might be misleading here because you install to an environment and then realise/reproduce/download the already installed packages for that environment on another machine. For brevity it's probably OK to imply that they won't be downloaded again if they are already downloaded. How does the following sound?

> Another developer on the same project can get started immediately with [`flox activate`][flox_activate], which will automatically download the same versions of those packages on their machine: